libs/lb/lb-rs/src/shared/account.rs

```diff
@@ -24,6 +29,96 @@ impl Account {
     pub fn public_key(&self) -> PublicKey {
         PublicKey::from_secret_key(&self.private_key)
     }
+
+    pub fn get_phrase(&self) -> SharedResult<[String; 24]> {
+        let key = self.private_key.serialize();
+        let key_bits = key.iter().fold(String::new(), |mut out, byte| {
+            let _ = write!(out, "{:08b}", byte);
```

Comment:
If I understand this pipeline correctly you're going from: `[u8]` -> `String` -> `u16`. 

This is probably not the right approach, including the checksum. 

Working with bytes within db_rs and with crypto stuff I would expect to see you use one of:
- https://doc.rust-lang.org/std/primitive.u16.html#method.from_be_bytes
- https://doc.rust-lang.org/std/primitive.u16.html#method.from_le_bytes

to perform the `[u8]` to `u16` conversion.

I think the resulting implementation will be much clearer to read. And this is some of the most critical code to be as clear as possible. Everything that we do rests on this. 

Also we do want this operation to be as fast as possible for things like the fuzzer (although I can just use the compact key so possibly a moot point).

Reply:
I now understand that the algorithm requires us to take 11 bits at a time which is not as convenient as I initially expected.